src/core/slash-commands/kilo.ts

```diff
@@ -36,74 +36,47 @@ export async function parseKiloSlashCommands(
 	}
 
 	// this currently allows matching prepended whitespace prior to /slash-command
-	const tagPatterns = [
-		{ tag: "task", regex: /<task>(\s*\/([a-zA-Z0-9_.-]+))(\s+.+?)?\s*<\/task>/is },
-		{ tag: "feedback", regex: /<feedback>(\s*\/([a-zA-Z0-9_.-]+))(\s+.+?)?\s*<\/feedback>/is },
-		{ tag: "answer", regex: /<answer>(\s*\/([a-zA-Z0-9_.-]+))(\s+.+?)?\s*<\/answer>/is },
-		{ tag: "user_message", regex: /<user_message>(\s*\/([a-zA-Z0-9_.-]+))(\s+.+?)?\s*<\/user_message>/is },
-	]
+	const tagPattern = /<(task|feedback|answer|user_message)>(\s*\/([a-zA-Z0-9_.-]+))(\s+.+?)?\s*<\/\1>/is
```

Comment:
Tbc, this is a refactor, but I'd like to change this regex, especially `(\s+.+?)?\s*`; what's happening there. I understand at least one space, but then optionally multiple points, and then possibly some more spaces?